core/refinerycms-core.gemspec

```diff
@@ -2,7 +2,7 @@
 require File.expand_path('../../core/lib/refinery/version', __FILE__)
 
 version = Refinery::Version.to_s
-rails_version = ['>= 4.2.3', '< 5.0']
+rails_version = ['>= 5.0.0', '< 5.1']
```

Comment:
After thinking about it, we should only support the latest version of Rails, not all versions. So I do not recommend to use Appraisals.